scripts/incentive-spreadsheet-registry.ts

```diff
@@ -21,7 +21,6 @@ export const FILES: { [ident: string]: IncentiveFile } = {
   },
   CO: {
     filepath: 'data/CO/incentives.json',
-    collectedFilepath: 'data/CO/collected.json',
```

Comment:
removed per conversation with @oyamauchi. Will add back in once we are ready to treat the json files as the source of truth.